src/tests/ut/clinic/clinic_test.cpp

```diff
@@ -7,7 +7,7 @@ TEST_F(ClinicTestFixture, GivenStaticContainerWhenDoctorCreatedExpectContainerSi
 {
     ASSERT_TRUE(Clinic::getDoctors().empty());
 
-    Doctor::createDoctor("Jan", "Pawel", "00000000000");
+    Doctor::createDoctor("Jan", "Pawel", "00000000000", Gender::Male);
```

Comment:
By the test we can see that validation is not performed on creation of the object. Please correct that. 
I know there's quite a lot of tests and you might need to find a few pesels to satisfy them, but it would show that our validation works as intented

Reply:
Done